tests/acceptance-tests/src/acceptance/git_buildpack_test.go

```diff
@@ -0,0 +1,29 @@
+package acceptance_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+)
+
+var _ = Describe("GitBuildpack", func() {
```

Comment:
I understand the underlying functionality is using git, but I think this might be more descriptive to us if it was called "custom buildpacks" (to match the manifest option and what we've been calling it).